css/style.css

```diff
@@ -16,4 +16,174 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General style */
+* {
+    margin: 0;
+    padding: 0;
+    /* background-color: white; */
+  }
+  
+  body {
+    width: 100%;
+  }
+  
+  
+  li,
```

Comment:
I don't think its necessary to style both li and a... as its only the a that needs to have the colour and text-decoration none.

Unless you meant that u want to target all anchor tags within list items only then it should have been `li a { `.